### PR TITLE
Avoid the use of 'integrate'

### PR DESCRIPTION
**Pull request by AlexJerabek**
The term "integrate" has [specific marketing meaning](https://microsoft.sharepoint.com/sites/celaweb-marketing/sitepages/marketing-and-advertising-content-red-flag-terms.aspx?xsdata=MDR8MDF8UGV0cmEuUm9uYWxkQG1pY3Jvc29mdC5jb218YTY1MzFlY2VhNTA0NDIxMTIyYjgwOGQ4MTkzY2ExMDB8NzJmOTg4YmY4NmYxNDFhZjkxYWIyZDdjZDAxMWRiNDd8MXwwfDYzNzI4NzA5NTQ5ODcyMDYyMXxVbmtub3dufFRXRnBiR1pzYjNkOGV5SldJam9pTUM0d0xqQXdNREFpTENKUUlqb2lWMmx1TXpJaUxDSkJUaUk2SWsxaGFXd2lMQ0pYVkNJNk1uMD18LTE%3D&sdata=OE5ZZDNXNHNUd3lZWGREQ0R0OWo5eUp3cnhkNERwSlJSWEo0V2lxVGg0OD0%3D), so this PR remove the use of the word.

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[13f5008](https://github.com/OfficeDev/office-scripts-docs/commits/13f5008252853568f3e34655ee1dde92d5d92dbb)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/power-automate-integration.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Integrate/docs/develop/power-automate-integration.md) | :white_check_mark:Succeeded | |
[docs/overview/excel.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Integrate/docs/overview/excel.md) | :white_check_mark:Succeeded | |
[docs/resources/vba-differences.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Integrate/docs/resources/vba-differences.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Integrate/docs/toc.yml) | :white_check_mark:Succeeded | |
[docs/tutorials/excel-power-automate-manual.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Integrate/docs/tutorials/excel-power-automate-manual.md) | :white_check_mark:Succeeded | |
[docs/tutorials/excel-power-automate-trigger.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Integrate/docs/tutorials/excel-power-automate-trigger.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C6%5C29%5C918d1c35-f3a4-631b-1edf-a1a2af22eee5%5CPullRequest%5C202006291614327769-129%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=bqENOpNOthYhdkCYFQ00xPETUaaNnUeK6ydk83fCsuo%3D&st=2020-06-29T16%3A10%3A52Z&se=2020-07-30T16%3A15%3A52Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/918d1c35-f3a4-631b-1edf-a1a2af22eee5?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/resources/vba-differences.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ This article describes the main differences between VBA macros (as well as VBA i
 
 ## Platform and ecosystem
 
-VBA is designed for the desktop and Office Scripts are designed for the web. VBA can interact with a user's desktop. This lets it integrate with similar technologies, such as COM and OLE. However, VBA has no convenient way to call out to the internet.
+VBA is designed for the desktop and Office Scripts are designed for the web. VBA can interact with a user's desktop. This lets it work with similar technologies, such as COM and OLE. However, VBA has no convenient way to call out to the internet.
```
Comment (mscharlock) [inline docs/resources/vba-differences.md:19]: I feel like "lets it work" is weird phrasing? I don't have a great alternative suggestion but maybe look at the previous sentence and see if there's some kind of combining that can happen between them

**Comment by o365devx** [comment]
Docs Build status updates of commit _[4ef6ca2](https://github.com/OfficeDev/office-scripts-docs/commits/4ef6ca287a6ee0ae560e1621e165e906ebe7779f)_: 

:clock10: **Pending**: waiting for processors ([11](https://ops.microsoft.com/#/statistic) builds ahead of you) 

 :warning: **Docs Build is busy**, currently there are 11 builds ahead of this one, for more information you can view the Build queue graph on the [Docs Portal](https://ops.microsoft.com/#/statistic).

**Comment by AlexJerabek** [comment]
Thanks @mscharlock. Updated based on your feedback.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[4ef6ca2](https://github.com/OfficeDev/office-scripts-docs/commits/4ef6ca287a6ee0ae560e1621e165e906ebe7779f)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/power-automate-integration.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Integrate/docs/develop/power-automate-integration.md) | :white_check_mark:Succeeded | |
[docs/overview/excel.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Integrate/docs/overview/excel.md) | :white_check_mark:Succeeded | |
[docs/resources/vba-differences.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Integrate/docs/resources/vba-differences.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Integrate/docs/toc.yml) | :white_check_mark:Succeeded | |
[docs/tutorials/excel-power-automate-manual.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Integrate/docs/tutorials/excel-power-automate-manual.md) | :white_check_mark:Succeeded | |
[docs/tutorials/excel-power-automate-trigger.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Integrate/docs/tutorials/excel-power-automate-trigger.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C6%5C30%5C918d1c35-f3a4-631b-1edf-a1a2af22eee5%5CPullRequest%5C202006302019445019-129%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=aPGaZkN2XHk5L4hc2pV2Q5%2FtlDkEYb1L%2BCdYWD2fCZ8%3D&st=2020-06-30T20%3A15%3A58Z&se=2020-07-31T20%3A20%3A58Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/918d1c35-f3a4-631b-1edf-a1a2af22eee5?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Pull request merged**